modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc

```diff
@@ -156,49 +153,48 @@ def ray_fn(openshift_server: str, openshift_token: str):
+if __name__ == '__main__': <14>
     from kfp.compiler import Compiler
     Compiler().compile(ray_integration, 'compiled-example.yaml')
 
 ----
-<1> Imports from the CodeFlare SDK the packages that define the cluster functions
-<2> Authenticates with the cluster by using values that you specify when creating the pipeline run;
-if your cluster uses self-signed certificates, include `ca-cert-path=<path>` in the `TokenAuthentication` parameter list, where `<path>` is the path to the cluster-wide Certificate Authority (CA) bundle that contains the self-signed certificates
-// Commenting out second part of callout 2 until RHOAIENG-880 is fixed
-//; you can omit this section if the Ray cluster is configured to use the same namespace as the data science project
-<3> Specifies the Ray cluster configuration: replace these example values with the values for your Ray cluster
-<4> Specifies the location of the Ray cluster image: if using a disconnected environment, replace the default value with the location for your environment
-<5> Specifies the local queue to which the Ray cluster will be submitted: you can omit this line if you configured a default local queue
-<6> Creates a Ray cluster using the specified image and configuration
-<7> Waits for the Ray cluster to be ready before proceeding
-<8> Enables the Ray client to connect to a secure Ray cluster that has mutual Transport Layer Security (mTLS) enabled; mTLS is enabled by default in the CodeFlare component in {productname-short}
-<9> Replace the example details in this section with the details for your workload
-<10> Removes the Ray cluster when your workload is finished
-<11> Replace the example name and description with the values for your workload
-<12> Compiles the Python code and saves the output in a YAML file
+<1> Imports Ray.
+<2> Imports the `time` package so that you can use the `sleep` function to wait during code execution, as a workaround for link:https://issues.redhat.com/browse/RHOAIENG-7346[RHOAIENG-7346].
+<3> Imports from the CodeFlare SDK the packages that define the cluster functions.
+<4> Specifies the Ray cluster configuration: replace these example values with the values for your Ray cluster.
+<5> Optional: Specifies the project where the Ray cluster is created. Replace the example value with the name of your project. If you omit this line, the Ray cluster is created in the current project.
+<6> Specifies the location of the Ray cluster image. If using a disconnected environment, replace the default value with the location for your environment.
+<7> Specifies the local queue to which the Ray cluster will be submitted. If a default local queue is configured, you can omit this line.
+<8> Creates a Ray cluster by using the specified image and configuration.
+<9> Waits for the Ray cluster to be ready before proceeding. As a workaround for link:https://issues.redhat.com/browse/RHOAIENG-7346[RHOAIENG-7346], use `time.sleep(180)` instead of `cluster.wait_ready()`.
```

Comment:
Could a better term be used than "to be ready"?

"To initialize"?

Reply:
I believe the `ray.init` command does the initialization.
This callout is for the original `cluster.wait_ready() ` command, so I think the current text is more accurate. I've rephrased slightly as follows:
" Waits until the Ray cluster is ready before proceeding."